test/generators/html/Ocamlary.html

```diff
@@ -2319,7 +2320,9 @@ <h4 id="advanced-type-stuff">
     <div class="spec type anchored" id="type-dep5">
      <a href="#type-dep5" class="anchor"></a>
      <code><span><span class="keyword">type</span> dep5</span>
-      <span> = <a href="Ocamlary-Dep13-class-c.html">Dep13.c</a></span>
+      <span> = 
+       <a href="Ocamlary-Dep11-module-type-S-class-c.html">Dep13.c</a>
+      </span>
```

Comment:
It's correct but slightly odd to link from `Dep13.c` to `Dep11.S.c` when there is a `Dep13` page containing `c`...